### PR TITLE
Utility functions for parsing config files and creating Graphs

### PR DESCRIPTION
**Pull request by Caw3**

### CODE REVIEW

File: src/utils/Graph.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
+/** Class representing a Graph */
```
Comment (JoXW100) [inline src/utils/Graph.js:1]: Filenames should start with lowercase characters

File: src/utils/Graph.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+   */
+  constructor () {
+    this.nodes = []
+    this.connections = []
+  }
+
+  /**
+    * @method
+    * @param {Node} node to be added
+    */
```
Comment (JoXW100) [inline src/utils/Graph.js:14]: Strange indentations of function descriptions (same with the other functions in this file)

File: src/utils/Graph.js [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+        row.to,
+        'linktext' in row ? row.linktext : null
+      )
+
+      graph.addConnection(connection)
+    }
+  })
+
+  return graph
+}
```
Comment (JoXW100) [inline src/utils/Graph.js:171]: Nothing exported? unused file?

File: src/utils/handleFiles.js [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+  })
+
+  let result = []
+  for (let i = 0; i < parsers.length; i++) {
+    const rowObjectArray = await parsers[i]
+    result = result.concat(rowObjectArray)
+  }
+
+  callback(result)
+}
```
Comment (JoXW100) [inline src/utils/handleFiles.js:158]: Nothing exported from this file?

File: package.json [hunk 5]
```diff
@@ -16,7 +16,8 @@
         "next": "12.1.0",
         "react": "17.0.2",
         "react-dom": "17.0.2",
-        "react-flow-renderer": "^10.0.6"
+        "react-flow-renderer": "^10.0.6",
+    	  "xlsx": "^0.18.5"
```
Comment (JoXW100) [inline package.json:20]: indentation error

**Review by JoXW100 — COMMENTED** [review]
Some general issues found

**Review by JoXW100 — APPROVED** [review]
Some overly complex and long functions but everything seams to be correct.

**Review by thomssonkarl — APPROVED** [review]

**Pull request merged**